src/plugins/graph/adornments/movable-line/movable-line.tsx

```diff
@@ -1,20 +1,21 @@
-import React, {useCallback, useEffect, useRef, useState} from "react";
+import React, {useCallback, useContext, useEffect, useRef, useState} from "react";
 import {autorun} from "mobx";
 import { observer } from "mobx-react-lite";
-import {drag, select} from "d3";
+import { drag, select, Selection} from "d3";
```

Comment:
```suggestion
import {drag, select, Selection} from "d3";
```
consistent spacing (at least within the line)